common/lib/xmodule/xmodule/data.py

```diff
@@ -0,0 +1,25 @@
+"""
+Public data structures for this app.
+
+See OEP-49 for details
+"""
+from enum import Enum
+
+
+class CertificatesDisplayBehaviors(str, Enum):
```

Comment:
I'm not familiar with this pattern. Why does this inherit from both `Enum` and `str`?

Reply:
It allows you to do string comparisons from stored values (e.g. the DB) without doing a `.value`. Suggested from a stack overflow post that pointed here: https://docs.python.org/3/library/enum.html#others
```
In [2]: class WithStr(str, Enum):
   ...:     END = "end"
   ...:     

In [3]: class WithoutStr(Enum):
   ...:     END = "end"
   ...:     

In [4]: "end" == WithStr.END
Out[4]: True

In [5]: "end" == WithoutStr.END
Out[5]: False
```

Reply:
Huh. TIL.